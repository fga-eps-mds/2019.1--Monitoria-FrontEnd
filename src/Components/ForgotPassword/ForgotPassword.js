import React, { Component } from 'react';
import { Grid, Button, TextField } from '@material-ui/core' ;
import logo from '../../Assets/img/Logo.png';
import './ForgotPassword.css';
import lightBlue from '@material-ui/core/colors/lightBlue';
import  {Link}  from 'react-router-dom';
import { MuiThemeProvider, createMuiTheme } from '@material-ui/core/styles';

const theme = createMuiTheme({
  palette: {
    primary: { main: lightBlue[50] },
    secondary: { main: '#11cb5f' },
  },
  typography: { useNextVariants: true },
});

class ForgotPassword extends Component {
  render() {
    return (
      <div className="ForgotPasswordBackground">
        <Grid container  alignContent="center" justify="center" direction="column" alignItems="center">
          <div style={{ padding: 80 }}>
      
      
        <Grid container  alignContent="center" justify="center" direction="column" alignItems="center" spacing="24">
          <img src={logo} alt="Logo" />
          
          <Grid item md-auto>
            <TextField
              id="emailTextField"
              label="Email"
              margin="normal"
              />
          </Grid>
          </Grid>
          <Grid container alignContent="center" justify="center" direction="column" spacing="16" alignItems="center" style={{marginTop: 25}}>
          <Grid item md-auto>
            <MuiThemeProvider theme={theme}>

              <Button component={Link} to="/ModifyPassword" variant="outlined" color="primary">
                Enviar
              </Button>
            </MuiThemeProvider>

              </Grid>
            <Grid item md-auto>
              <MuiThemeProvider theme={theme}>
                <Button component={Link} to="/" variant="outlined" color="primary">
                  Voltar
                </Button>
              </MuiThemeProvider>
            </Grid>
            
            </Grid>
            </div>
            </Grid>
            
        </div>
      
    );   
  }
}

export default ForgotPassword;
