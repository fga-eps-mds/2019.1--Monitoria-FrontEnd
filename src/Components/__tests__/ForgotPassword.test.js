import React from 'react';
import { shallow } from 'enzyme';
import ForgotPassword from '../ForgotPassword/ForgotPassword';
import toJson from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';
import Enzyme from 'enzyme';

Enzyme.configure({adapter: new Adapter()});



describe('Testing ForgotPassword component', () => {
    it('Test if ForgotPassword renders correctly', () =>{
        const tree = shallow(<ForgotPassword/>);
        expect(toJson(tree)).toMatchSnapshot();
    });
    
    
});