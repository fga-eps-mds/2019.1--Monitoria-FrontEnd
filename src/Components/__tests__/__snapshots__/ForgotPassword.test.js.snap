// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing ForgotPassword component Test if ForgotPassword renders correctly 1`] = `
<Route>
  <Component />
</Route>
`;
